### PR TITLE
Add a tooltip for frame tracks in Mizar

### PR DESCRIPTION
**Pull request by akopich**
The result is achieved via implementing a custom Model for 
the corresponding QComboBox. 

Tests: Unit, Manual
Bug: http://b/242841057

### CODE REVIEW

File: src/MizarModels/include/MizarModels/FrameTrackListModel.h [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        return MakeTooltip(id, name);
+      case kFrameTrackIdRole:
+        return QVariant::fromValue(id);
+      default:
+        return {};
+    }
+  }
+
+ private:
+  static std::vector<std::pair<FrameTrackId, std::string>> MakeDisplayedNames(
```
Comment (vickyliu-go4it) [inline src/MizarModels/include/MizarModels/FrameTrackListModel.h:67]: Nit: nodiscard?

File: src/MizarModels/include/MizarModels/FrameTrackListModel.h [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+    orbit_base::sort(std::begin(id_to_displayed_name), std::end(id_to_displayed_name),
+                     &std::pair<FrameTrackId, std::string>::second);
+    return id_to_displayed_name;
+  }
+
+  [[nodiscard]] static std::string MakeFrameTrackString(
+      const orbit_client_data::ScopeInfo& scope_info) {
+    const std::string_view type_string =
+        scope_info.GetType() == orbit_client_data::ScopeType::kDynamicallyInstrumentedFunction
+            ? "  D"
```
Comment (vickyliu-go4it) [inline src/MizarModels/include/MizarModels/FrameTrackListModel.h:88]: Nit: maybe define some constants for this, which will also benefit the unit tests

**Review by vickyliu-go4it — APPROVED** [review]
Nice change! Thanks

File: src/MizarModels/include/MizarModels/FrameTrackListModel.h (hunk 1 above)
Comment (akopich) [reply src/MizarModels/include/MizarModels/FrameTrackListModel.h:67]: Yep. Also, I replace `std::pair` with a struct.

File: src/MizarModels/include/MizarModels/FrameTrackListModel.h (hunk 2 above)
Comment (akopich) [reply src/MizarModels/include/MizarModels/FrameTrackListModel.h:88]: But we would have to expose the constants and write some more code in the test glueing together these prefixes and the names (for now I've just got this hard-coded in the test and IMO it's fine).

**Pull request merged**